testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/PointsActivity.kt

```diff
@@ -67,10 +67,19 @@ class PointsActivity : BaseActivity() {
         if (pointTimeLimited == "") pointTimeLimited = "0"
         var pointRakutenCash = edtPointRakutenCash.text.toString()
         if (pointRakutenCash == "") pointRakutenCash = "0"
-        settings.points = Points(
-                pointStandard.toInt(),
-                pointTimeLimited.toInt(),
-                pointRakutenCash.toInt()
-        )
+        try {
+            settings.points = Points(
+                    pointStandard.toInt(),
+                    pointTimeLimited.toInt(),
+                    pointRakutenCash.toInt()
+            )
+        } catch (e: NumberFormatException) {
+            e.printStackTrace()
+        }
```

Comment:
Instead of having one try/catch, you should have method which takes a String as parameter and returns an Int, it could return the int that it passed or `-1` if it caught an exception.

```suggestion
       settings.points = Points(
                extractPointsFrom(pointStandard),
                extractPointsFrom(pointTimeLimited),
                extractPointsFrom(pointRakutenCash)
        )
```
```kotlin
fun extractPointsFrom(pointString: String): Integer {
    try {
          return pointString.toInt()
      } catch (e: NumberFormatException) {
          return -1
      }
}
```

Reply:
@Climbatize Oh, sorry I missed your suggestion before merging 🙇 
But, I think, we can return `0` when there is exception. 
Although, because of `maxLength="9"` in the input UI, we will not have any exception now. So, I'll think about this suggestion in a separate PR. Thank you!